apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/aop/NamespaceAcquireLockAspect.java

```diff
@@ -117,7 +117,7 @@ void acquireLock(long namespaceId, String currentUser) {
 
   private void acquireLock(Namespace namespace, String currentUser) {
     if (namespace == null) {
-      throw new BadRequestException("namespace not exist.");
+      throw BadRequestException.namespaceNotExists();
```

Comment:
Shall we also pass in the namespace info?

Reply:
Since we cannot obtain the namespace information from the method's definition, we cannot print it in the exception logs. This situation also occurs in other places.